diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -662,7 +662,8 @@ config("compiler") {
     # TODO(crbug.com/376278218): This causes segfault on Linux ARM builds.
     # It also causes segfault on Linux s390x:
     # https://github.com/llvm/llvm-project/issues/149511
-    if (is_linux && use_lld && current_cpu != "arm" && current_cpu != "s390x") {
+    if (is_linux && use_lld && current_cpu != "arm" &&
+        current_cpu != "arm64" && current_cpu != "s390x") {
       cflags += [ "-Wa,--crel,--allow-experimental-crel" ]
     }
   }
diff --git a/rtc_base/copy_on_write_buffer.cc b/rtc_base/copy_on_write_buffer.cc
--- a/rtc_base/copy_on_write_buffer.cc
+++ b/rtc_base/copy_on_write_buffer.cc
@@ -28,6 +28,6 @@ namespace webrtc {
 
 CopyOnWriteBuffer::RawBuffer::RawBuffer(size_t size)
-    : size_(size), data_(std::make_unique_for_overwrite<uint8_t[]>(size)) {}
+    : size_(size), data_(std::unique_ptr<uint8_t[]>(new uint8_t[size])) {}
 
 scoped_refptr<CopyOnWriteBuffer::RefCountedBuffer>
 CopyOnWriteBuffer::CreateBuffer(size_t capacity) {
diff --git a/call/payload_type_picker.cc b/call/payload_type_picker.cc
--- a/call/payload_type_picker.cc
+++ b/call/payload_type_picker.cc
@@ -368,6 +368,6 @@ RTCError RtpHeaderExtensionRecorder::AddMapping(RtpHeaderExtensionId id,
                                                 absl::string_view uri,
                                                 bool encrypt) {
-  auto it = uri_to_id_.find(std::pair{uri, encrypt});
+  auto it = uri_to_id_.find(std::pair{std::string(uri), encrypt});
   if (it != uri_to_id_.end()) {
     if (it->second != id) {
       RTC_HISTOGRAM_BOOLEAN(
@@ -391,6 +391,6 @@ RTCErrorOr<RtpHeaderExtensionId> RtpHeaderExtensionRecorder::LookupId(
     absl::string_view uri,
     bool encrypt) const {
-  auto it = uri_to_id_.find(std::pair{uri, encrypt});
+  auto it = uri_to_id_.find(std::pair{std::string(uri), encrypt});
   if (it == uri_to_id_.end()) {
     return RTCError(RTCErrorType::INVALID_PARAMETER,
                     "No ID found for extension");
diff --git a/rtc_base/ssl_stream_adapter.h b/rtc_base/ssl_stream_adapter.h
--- a/rtc_base/ssl_stream_adapter.h
+++ b/rtc_base/ssl_stream_adapter.h
@@ -129,6 +129,6 @@ class SSLStreamAdapter : public StreamInterface {
   static std::unique_ptr<SSLStreamAdapter> Create(
       std::unique_ptr<StreamInterface> stream,
       absl::AnyInvocable<void(SSLHandshakeError)> handshake_error,
-      nullptr_t /*field_trials*/) {
+      std::nullptr_t /*field_trials*/) {
     return Create(std::move(stream), std::move(handshake_error));
   }
